src/detect-engine-build.c

```diff
@@ -1129,15 +1130,160 @@ int CreateGroupedPortList(DetectEngineCtx *de_ctx, DetectPort *port_list, Detect
         uint32_t unique_groups, int (*CompareFunc)(DetectPort *, DetectPort *));
 int CreateGroupedPortListCmpCnt(DetectPort *a, DetectPort *b);
 
+#define MAX_PORT_VAL 65536
```

Comment:
max value is 65535, right? Or UINT16_MAX (which is the same)

Reply:
well the name is prob wrong, I wanted to use this to allocate an array of size 65536, which I could have done by writing `UINT16_MAX + 1` 🤦🏽‍♀️